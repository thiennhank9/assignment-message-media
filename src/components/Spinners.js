import Spinner from "react-bootstrap/Spinner";

export default function Spinners() {
  return (
    <div className="d-flex my-3 flex-grow-1 justify-content-center align-items-center">
      <Spinner animation="grow" variant="secondary" />
      <Spinner animation="grow" variant="success" />
      <Spinner animation="grow" variant="danger" />
      <Spinner animation="grow" variant="warning" />
      <Spinner animation="grow" variant="info" />
      <Spinner animation="grow" variant="light" />
      <Spinner animation="grow" variant="dark" />
    </div>
  );
}
