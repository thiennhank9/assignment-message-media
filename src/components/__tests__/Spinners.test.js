import renderer from "react-test-renderer";
import Spinners from "../Spinners";

test("Spinners - render correctly", () => {
  const tree = renderer.create(<Spinners />).toJSON();
  expect(tree).toMatchSnapshot();
});
