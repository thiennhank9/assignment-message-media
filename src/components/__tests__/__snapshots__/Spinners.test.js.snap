// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinners - render correctly 1`] = `
<div
  className="d-flex my-3 flex-grow-1 justify-content-center align-items-center"
>
  <div
    className="spinner-grow text-secondary"
  />
  <div
    className="spinner-grow text-success"
  />
  <div
    className="spinner-grow text-danger"
  />
  <div
    className="spinner-grow text-warning"
  />
  <div
    className="spinner-grow text-info"
  />
  <div
    className="spinner-grow text-light"
  />
  <div
    className="spinner-grow text-dark"
  />
</div>
`;
